### PR TITLE
FailoverClusters BulkFixup (2/4) - Text formatting updates

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk formatting checklist update before applying further quality updates:
- cmdlet name - Change bolds to code spans
- cmdlet/vars - Change bar ecmdlet/var to code spans
- parameters - change italic formatting to bold
- parameters - change code span formatting to bold
- change italics to use underscores
- change bold-underscores to asterisk
- change bold keystrokes to kbd
- Use contractions
- Fix indented code blocks
- Find bad syntax reflows

This will be followed by separate PRs to update example formatting before content updates

### CODE REVIEW

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Virtual Machine1           node2                              Online
 ```
 
 This example performs a live migration of the clustered virtual machine named Virtual Machine1 to
-the node named node2. The Windows PowerShell® prompt does not return until the action is complete.
+the node named node2. The Windows PowerShell® prompt doesn't return until the action is complete.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:48]:
```suggestion
the node named node2. The Windows PowerShell prompt doesn't return until the action is complete.
```

File: docset/winserver2022-ps/failoverclusters/New-ClusterFaultDomain.md [hunk 2]
```diff
@@ -124,10 +124,10 @@ Specifies the type of the fault domain that this cmdlet creates.
...
 The acceptable values for this parameter are:
 
-- Site
-- Rack
-- Chassis
-- Node
+- **Site**
+- **Rack**
+- **Chassis**
+- **Node**
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/New-ClusterFaultDomain.md:130]: Specific values for parameters should be formatted as code.
```suggestion
- `Site`
- `Rack`
- `Chassis`
- `Node`

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Minor notes. Otherwise, LGTM!

**Comment by robinharwood** [comment]
Thanks for the review @michaeltlombardi - I've applied your suggestions, should be good to go. I think I must have had the parameter formatting for the acceptable values the wrong way round in my head! Thanks again

**Pull request merged**